infra/RepoManager.py

```diff
@@ -29,14 +29,7 @@
 
 class RepoManagerError(Exception):
   """Class to describe the exceptions in RepoManager."""
-
-  def __init__(self, message):
-    """ Init the exception class.
-
-    Args:
-      message: the message to propagate to user
-    """
-    super().__init__(message)
+  pass
```

Comment:
nit: Not necessary (I think the linter would have told you this, did it?)

Reply:
Your right, I forgot to lint it. I now have.